### PR TITLE
Create tab

### PR DESCRIPTION
**Pull request by saucekode**
Before submitting a final PR, please:

- [ ] Run `npm test` on your machine
- [ ] Run `npm run format` to keep code formatted consistently
- [ ] Reference the issue you are fixing in at least one of your commit messages, as `Fixes #X`

### CODE REVIEW

**Comment by saucekode** [comment]
@ianb please take a look at 

https://github.com/mozilla/firefox-voice/pull/1370/commits/b32b4c3eba14e57784d6836f72499c90585a7e0d

It's a better fix for #1258 and @alexandra-martin pointed out this bug and another for a confirm modal to the user as stated by her here #1319. Though someone suggested I create a contentScript since the extension isn't seen by the DOM(hope that's correct). What's a better option? Thank you.

File: extension/intents/window/window.js [hunk 1]
```diff
@@ -48,16 +48,9 @@ intentRunner.registerIntent({
...
-    // getAll normal window
-    const gettingAll = await browser.windows.getAll({
-      windowTypes: ["normal"],
-    });
-    // find target windowId
-    const targetWindowId = findTargetWindowId(gettingAll, currentWindowId);
-    await browser.windows.remove(targetWindowId);
+    context.keepPopup();
+    const currentWindow = await browser.windows.getCurrent();
+    await browser.windows.remove(currentWindow.id);
```
Comment (ianb) [inline extension/intents/window/window.js]: I think this change isn't intended to be part of the PR...?

Comment (saucekode) [reply extension/intents/window/window.js]: Hello @ianb yes it isn't.

**Comment by ianb** [comment]
Hmm... I'm a little confused, the PR has some unrelated changes, but I suppose all the changes are correct so I can merge them.

**Pull request merged**